components/Portfolio/Insurance/Deposits/ActiveDeposits/index.tsx

```diff
@@ -32,26 +34,32 @@ const ActiveDeposits: FC<ADProps> = ({ parentHeight }: ADProps) => {
                     ))}
                 </TableHeader>
                 <TableBody>
-                    {Object.values(activeDeposits).map((tracer, i) => {
+                    {Object.values(insuranceContracts).map((insuranceContract, i) => {
                         const show = showButton === i;
                         return (
                             <TableRow
                                 key={`table-row-${i}`}
                                 onMouseEnter={() => setShowButton(i)}
                                 onMouseLeave={() => setShowButton(-1)}
                             >
-                                <TableCell>{tracer.market}</TableCell>
-                                <TableCell>{tracer.realisedAPY}</TableCell>
-                                <TableCell>{tracer.ownership}</TableCell>
-                                <TableCell>{tracer.unrealisedValue}</TableCell>
+                                <TableCell>{insuranceContract.market}</TableCell>
+                                <TableCell>{toPercent(insuranceContract.apy.toNumber())}</TableCell>
+                                <TableCell>{insuranceContract.liquidity.toNumber()}</TableCell>
```

Comment:
If this is the users balance I think there is a userBalance key

Reply:
sure weird but userBalance is always empty while liquidity gets the balance amount